reactioncommerce/reaction
# Migration updates to fix Localization, uol, and paymentMethod errors on 1.0 > 1.5

**Fixes:**
1. Reported error on Localization dashboard [in comments](https://github.com/reactioncommerce/reaction/issues/2938#issuecomment-333249832) and #2965 
2. Fixes errors with "Units of length" when creating new shops
3. Updates shipping migration with new status (picked and labeled - introduced in #2613)
4. Error on refunds (an order with captured payment (at <1.4 Reaction), when migrated to marketplace/release1.5 branch, throws an error if you try to refund (caused by changes introduced in #2022) 

### How to test
1. Follow instructions in #2962 about creating orders in v 1.0 and moving back to marketplace version. While in v1.0 capture payment in one of the orders (we'll use it to test refund later)
2. After moving from <1.4 to 1.5, confirm that:
  * Localization dashboard works. No errors should be thrown when it opened
  * Creating new shops work fine without any Units of Length errors.
3. In the Order dashboard, visit the order with captured payment and try to do a refund. You should get a success message and no errors.
